resolver/addressable_resolver_test.go

```diff
@@ -100,7 +101,7 @@ func TestGetURI_ObjectReference(t *testing.T) {
 				DeprecatedAPIVersion: addressableAPIVersion,
 				DeprecatedNamespace:  testNS,
 			},
-			wantErr: fmt.Errorf("ref and [apiVersion, kind, name] can't be both present"),
+			wantErr: fmt.Sprintf("ref and [apiVersion, kind, name] can't be both present"),
```

Comment:
```suggestion
			wantErr: "ref and [apiVersion, kind, name] can't be both present",
```
And same below where there's no formatting.